### PR TITLE
react-big-calendar: Add missing prop into Components interface

### PR DESCRIPTION
**Pull request by lksilva**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

### CODE REVIEW

**Review by eps1lon — COMMENTED** [review]
Could you add a link to the docs?

**Comment by lksilva** [comment]
Sorry but i don't understand, but my PR is for fix type for a new feature  https://github.com/intljusticemission/react-big-calendar/commit/6708a4583a2179112dbc9c999a68161bb46de186

File: types/react-big-calendar/index.d.ts [hunk 1]
```diff
@@ -148,6 +149,7 @@ export interface Components<TEvent extends Event = Event> {
      * component used as a header for each column in the TimeGridHeader
      */
     header?: React.ComponentType<HeaderProps>;
+    resourceHeader?: React.SFC | React.Component | React.ComponentClass | JSX.Element;
```
Comment (eps1lon) [inline types/react-big-calendar/index.d.ts]: Would be nice if we could follow the same pattern as for the `header` prop i.e.
```suggestion
    resourceHeader?: React.ComponentType<ResourceHeaderProps>;
```

where `ResourceHeaderProps` matches https://github.com/intljusticemission/react-big-calendar/blob/9332ab72ae84334793b390833df8b32e88a2f1c4/src/ResourceHeader.js#L8-L12

**Review by eps1lon — CHANGES_REQUESTED** [review]

File: types/react-big-calendar/index.d.ts (hunk 1 above)
Comment (lksilva) [reply types/react-big-calendar/index.d.ts]: Sorry but this is my first contribution and I'm learning how open source world works! Thanks for the remark and I will fix it.

Comment (eps1lon) [reply types/react-big-calendar/index.d.ts]: No need to apologize. That's what review is for.

File: types/react-big-calendar/index.d.ts [hunk 2]
```diff
@@ -116,6 +117,12 @@ export interface HeaderProps {
     localizer: DateLocalizer;
 }
 
+export interface ResourceHeaderProps {
+    label: string;
```
Comment (eps1lon) [inline types/react-big-calendar/index.d.ts]:
```suggestion
    label: React.ReactNode;
```

is equivalent to `PropTypes.node`. It does include `string`

**Review by eps1lon — CHANGES_REQUESTED** [review]
Almost done. Just one minor thing left.

**Review by eps1lon — APPROVED** [review]

**Comment by uniqueiniquity** [comment]
Congratulations on your first DefinitelyTyped contribution!
Thank you for being a part of the community!

**Pull request merged**